### ISSUE TITLE
Grid: Cell order first and last do not work as expected

### ISSUE THREAD

**Issue by spraguey** [Formstone/Formstone#243]
```
<div class="fs-row">
    <div class="fs-cell fs-all-third">
        <span class="label">1</span>
    </div>
    <div class="fs-cell fs-all-third">
        <span class="label">2</span>
    </div>
    <div class="fs-cell fs-all-third fs-first highlight">
        <span class="label">3</span>
    </div>
</div>
```

The above code is from the demo page for Formstone's grid (https://formstone.it/components/grid/demo/).

The cell labeled 3 should be appearing first in the row (fs-first). It is not.

The same problem occurs with fs-last. On the demo page, that cell does not appear last.

Tested in current releases of Firefox and Chrome.

**Comment by benplum** [comment]
Something looks off - we'll check it out!

**Comment by benplum** [comment]
Fixed in 1.4.3!
